src/ts/spatialnavigation/navigationalgorithm.ts

```diff
@@ -59,9 +59,20 @@ function distance(a: Vector, b: Vector): number {
 function getElementVector(element: HTMLElement): Vector {
   const boundingRect = element.getBoundingClientRect();
 
+  let _x: number;
+  let _y: number;
+
+  if (boundingRect.hasOwnProperty('x') && boundingRect.hasOwnProperty('y')) {
+    _x = boundingRect.x;
+    _y = boundingRect.y;
+  } else {
+    _x = boundingRect.left;
+    _y = boundingRect.top;
+  }
```

Comment:
Seeing that we use that code in 2 places, can we move it into a separate function that we can use in both of them?

Reply:
agreed - added exported function `getBoundingRectFromElement` in latest commit